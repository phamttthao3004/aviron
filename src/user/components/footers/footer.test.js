import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Footer from './footer';

describe('Footer', () => {
    it('Render Footer component', () => {
        const com = shallow(<Footer />);
        expect(shallowToJson(com)).toMatchSnapshot();
    });
});
