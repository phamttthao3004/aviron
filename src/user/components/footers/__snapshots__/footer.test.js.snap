// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Render Footer component 1`] = `
<footer>
  <div
    className="container"
  >
    <div
      className="top col-md-12"
    >
      <img
        alt="Aviron logo"
        className="logo"
        src="/images/user/Logo-2.png"
      />
      <ul
        className="social"
      >
        <li>
          <a
            href="https://avironactive.com/"
          >
            <i
              className=" icon-twitter"
            />
          </a>
        </li>
        <li>
          <a
            href="https://avironactive.com/"
          >
            <i
              className=" icon-gplus"
            />
          </a>
        </li>
        <li>
          <a
            href="https://avironactive.com/"
          >
            <i
              className=" icon-instagram"
            />
          </a>
        </li>
        <li>
          <a
            href="https://avironactive.com/"
          >
            <i
              className=" icon-linkedin"
            />
          </a>
        </li>
        <li>
          <a
            href="https://avironactive.com/"
          >
            <i
              className=" icon-facebook"
            />
          </a>
        </li>
      </ul>
    </div>
    <div
      className="end col-md-12"
    >
      2019 Â© Power by Aviron Interactive Inc.
    </div>
  </div>
</footer>
`;
